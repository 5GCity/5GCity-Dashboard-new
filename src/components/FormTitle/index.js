/**
 * FormTitle Component
 * Please write a description
 *
 * @author Your Name <youremail@ubiwhere.com>
 */
import React from 'react'
import styled from 'styled-components'

export default ({ children, ...props }) => (
  <Wrapper {...props}>
  <Title>{props.title}</Title>
  <Border />
  </Wrapper>
)

const Wrapper = styled.div`
`

const Title = styled.p`
  margin-left: 8px;
  color: #EFF2F7;
  font-family: ${({ theme }) => theme.fontDin };
  text-transform: uppercase;
  font-size: 20px;
  letter-spacing: 0.5px;
  line-height: 24px;
  font-weight: bold;
  margin-bottom: 20px;
`
const Border = styled.p`
  border-bottom: 1px solid rgba(239,242,247,0.1);
`
