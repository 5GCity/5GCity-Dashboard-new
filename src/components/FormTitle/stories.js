/**
 * FormTitle Component Stories
 * Please write a description or remove this line
 *
 * @author Your Name <youremail@ubiwhere.com>
 */
import React from 'react'
import { storiesOf } from '@storybook/react'

import FormTitle from './index'

const State1 = () => (
  <FormTitle title={'genral info'} />
)

storiesOf('FormTitle', module)
  .add('primary', State1)
