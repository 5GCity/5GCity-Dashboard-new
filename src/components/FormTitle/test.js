/**
 * FormTitle Component Tests
 *
 * @author Your Name <youremail@ubiwhere.com>
 */
import React from 'react'
import renderer from 'react-test-renderer'
import FormTitle from './index'
import 'jest-styled-components'

it('renders correctly', () => {
  const tree = renderer
    .create(<FormTitle />)
    .toJSON()

  expect(tree).toMatchSnapshot()
})
