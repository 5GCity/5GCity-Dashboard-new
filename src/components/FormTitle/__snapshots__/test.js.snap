// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  margin-left: 8px;
  color: #EFF2F7;
  text-transform: uppercase;
  font-size: 20px;
  -webkit-letter-spacing: 0.5px;
  -moz-letter-spacing: 0.5px;
  -ms-letter-spacing: 0.5px;
  letter-spacing: 0.5px;
  line-height: 24px;
  font-weight: bold;
  margin-bottom: 20px;
}

.c1 {
  border-bottom: 1px solid rgba(239,242,247,0.1);
}

<div
  className=""
>
  <p
    className="c0"
  />
  <p
    className="c1"
  />
</div>
`;
